client/scss/components/_breadcrumbs.scss

```diff
@@ -1,4 +1,4 @@
-.page-explorer .w-breadcrumbs {
+.w-breadcrumbs:not(.editor-view .w-breadcrumbs) {
```

Comment:
From commit message:

We want to make the slim_header.html as the basis for the modern header
template that will be used everywhere, eventually replacing header.html.

Rather than making the page explorer be the special case where we want
the last item to be bigger, do it the other way around i.e. use the
smaller font size on the create/edit views. This ensures that the big
font size is used on all other views e.g. Inspect view.